### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by eric-radiant** [nielsgl/sequelize-paper-trail#17]

**Comment by nielsgl** [comment]
No, just haven't had time to resolved the issues, feel free to send a PR and I'd be happy to merge it 👍

**Comment by eric-radiant** [comment]
Glad to hear it isn't dead, sadly I haven't the time to track down issues either.  Was trying to evaluate if it would be useful for us.  Probably not until bugs are fixed and I can get which internal user was responsible for a revision.  Thanks for responding though, it looks like a slick project when it works.

**Comment by lijoantony** [comment]
It does work! I am using it in my (internal) projects.
Documentation is not sufficient though.
User Tracking has been added to the README now.
@nielsgl This issue may be closed.

**Comment by nielsgl** [comment]
Resolved with updated instructions in the README of version 1.3.3.  @eric-radiant if you're still having problems please create a new ticket.
